<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Infobar</class>
 <widget class="QWidget" name="Infobar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1122</width>
    <height>183</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: black;
border-color: black;</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>1</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>1</number>
     </property>
     <item>
      <widget class="QGroupBox" name="rundenzeit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>400</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
background: rgb(46, 52, 54);
border-color: black;</string>
       </property>
       <property name="title">
        <string>Verbleibende Rundenzeit</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLCDNumber" name="timer">
            <property name="smallDecimalPoint">
             <bool>false</bool>
            </property>
            <property name="segmentStyle">
             <enum>QLCDNumber::Flat</enum>
            </property>
            <property name="value" stdset="0">
             <double>60.000000000000000</double>
            </property>
            <property name="intValue" stdset="0">
             <number>60</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="weiter">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>30</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: green;
border: none;
border-radius: 6px;</string>
            </property>
            <property name="text">
             <string>Weiter</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="aufgeben">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>30</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(164, 0, 0);
border: none;
border-radius: 6px;</string>
            </property>
            <property name="text">
             <string>Aufgeben</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="planet">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
background: rgb(46, 52, 54);
border-color: black;</string>
       </property>
       <property name="title">
        <string/>
       </property>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="1" column="0">
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="7" column="0">
           <widget class="QLabel" name="minenanzahl">
            <property name="text">
             <string>Minenanzahl</string>
            </property>
           </widget>
          </item>
          <item row="10" column="0">
           <widget class="QPushButton" name="mine_bauen">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>30</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: blue;
border: none;
border-radius: 6px;</string>
            </property>
            <property name="text">
             <string>Mine bauen</string>
            </property>
           </widget>
          </item>
          <item row="8" column="1">
           <widget class="QLabel" name="erzvorkommen_value">
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
          <item row="8" column="0">
           <widget class="QLabel" name="minenanzahl_value">
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
          <item row="9" column="2">
           <widget class="QPushButton" name="transporter_bauen">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>30</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: blue;
border: none;
border-radius: 6px;</string>
            </property>
            <property name="text">
             <string>Transporter bauen</string>
            </property>
           </widget>
          </item>
          <item row="10" column="1">
           <widget class="QPushButton" name="werft_bauen">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>30</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: blue;
border: none;
border-radius: 6px;</string>
            </property>
            <property name="text">
             <string>Werft bauen</string>
            </property>
           </widget>
          </item>
          <item row="7" column="1">
           <widget class="QLabel" name="erzvorkommen">
            <property name="text">
             <string>Erzvorkommen</string>
            </property>
           </widget>
          </item>
          <item row="8" column="2">
           <widget class="QLabel" name="erzlager_value">
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
          <item row="10" column="2">
           <widget class="QPushButton" name="kampfschiff_bauen">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>30</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: blue;
border: none;
border-radius: 6px;</string>
            </property>
            <property name="text">
             <string>Kampfschiff bauen</string>
            </property>
           </widget>
          </item>
          <item row="7" column="2">
           <widget class="QLabel" name="erzlager">
            <property name="text">
             <string>Gelagertes Erz</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QLabel" name="planetname">
            <property name="text">
             <string>Planetname</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="resourcen">
       <property name="maximumSize">
        <size>
         <width>400</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
background: rgb(46, 52, 54);
border-color: black;</string>
       </property>
       <property name="title">
        <string>Resourcen</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_6">
          <item row="5" column="0">
           <widget class="QLabel" name="kampfschiffe">
            <property name="text">
             <string>Kampfschiffe</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="transporter">
            <property name="text">
             <string>Transporter</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLabel" name="transporter_value">
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QLabel" name="kampschiffe_value">
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="ertrag">
            <property name="text">
             <string>Ertrag/Runde</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLabel" name="ertrag_value">
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="erz">
            <property name="text">
             <string>Erz</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="erz_value">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>weiter</tabstop>
  <tabstop>aufgeben</tabstop>
  <tabstop>mine_bauen</tabstop>
  <tabstop>werft_bauen</tabstop>
  <tabstop>transporter_bauen</tabstop>
  <tabstop>kampfschiff_bauen</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
